### PR TITLE
chore(docs) updating insert-data page with csv import info

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: docs/develop/insert-data.md [hunk 1]
```diff
@@ -6,23 +6,20 @@ import TabItem from "@theme/TabItem"
...
-ingestion method in QuestDB and is recommended for high-performance
-applications.
+QuestDB supports large CSV import via SQL `COPY` statements and `INSERT` statements for ad hoc row ingestion. Both can be accessed using the embedded [Web Console](/docs/develop/web-console).
+
+For smaller CSV file import, you can use [HTTP REST API](#http-rest-api).
 
-For operational (ad-hoc) data ingestion the [Web Console](#web-console) makes it
-easy to upload CSV files and insert via SQL statements. You can also perform
-these same actions via the [HTTP REST API](#http-rest-api).
+[InfluxDB Line Protocol](#influxdb-line-protocol) is the recommended primary ingestion method in QuestDB for high-performance
```
Comment (amyshwang) [inline docs/develop/insert-data.md:13]: Sequence:
- ILP
- Posrgres
- COPY
- REST API

File: docs/develop/insert-data.md [hunk 2]
```diff
@@ -6,23 +6,20 @@ import TabItem from "@theme/TabItem"
...
-[PostgreSQL wire protocol](#postgresql-wire-protocol) as it provides
-parameterized querys which avoid SQL injection issues.
+[PostgreSQL wire protocol](#postgresql-wire-protocol) is less performant but offers more feedback. It is better suited for applications inserting via SQL programmatically as it provides parameterized queries which avoid SQL injection issues.
 
 In summary, these are the different options:
 
-- [Web Console](#web-console)
-  - CSV upload.
-  - SQL `INSERT` statements.
+- SQL keywords
```
Comment (amyshwang) [inline docs/develop/insert-data.md:20]: No. 3

File: docs/develop/insert-data.md [hunk 3]
```diff
@@ -6,23 +6,20 @@ import TabItem from "@theme/TabItem"
 This page shows how to insert data into QuestDB using different programming
 languages and tools.
 
-[InfluxDB Line Protocol](#influxdb-line-protocol) is the recommended primary
-ingestion method in QuestDB and is recommended for high-performance
-applications.
+QuestDB supports large CSV import via SQL `COPY` statements and `INSERT` statements for ad hoc row ingestion. Both can be accessed using the embedded [Web Console](/docs/develop/web-console).
```
Comment (amyshwang) [inline docs/develop/insert-data.md:9]: ```QuestDB supports large CSV import via SQL COPY statements. For smaller CSV file import, you can use HTTP REST API. ```

File: docs/develop/insert-data.md [hunk 4]
```diff
@@ -37,11 +34,33 @@ In summary, these are the different options:
   - SQL `INSERT` statements.
   - Use `curl` on the command line.
 
-## Web Console
+## SQL keywords
```
Comment (amyshwang) [inline docs/develop/insert-data.md:37]: Change back to Web console.

File: docs/develop/insert-data.md [hunk 5]
```diff
@@ -37,11 +34,33 @@ In summary, these are the different options:
   - SQL `INSERT` statements.
```
Comment (amyshwang) [inline docs/develop/insert-data.md:34]: - CSV upload.
  - SQL `INSERT` statements.

To 
- Small CSV import

File: docs/develop/insert-data.md (hunk 2 above)
Comment (amyshwang) [inline docs/develop/insert-data.md:20]:
```suggestion
- Web console
```

Reverting back to use interface as the section title.

**Comment by amyshwang** [comment]
We have another discussion about this page. Will open a new PR as this one has too many wrong changes.
